packages/react-router/__tests__/useLocation-test.tsx

```diff
@@ -14,16 +14,55 @@ describe("useLocation", () => {
       renderer = TestRenderer.create(
         <MemoryRouter initialEntries={["/home?the=search#the-hash"]}>
           <Routes>
-            <Route path="/home" element={<ShowPath />} />
+            <Route path="/home" element={<ShowLocation />} />
           </Routes>
         </MemoryRouter>
       );
     });
 
     expect(renderer.toJSON()).toMatchInlineSnapshot(`
       <pre>
-        {"pathname":"/home","search":"?the=search","hash":"#the-hash"}
+        {"pathname":"/home","search":"?the=search","hash":"#the-hash","state":null,"key":"default"}
       </pre>
     `);
   });
+
+  it("returns the scoped location object when nested in <Routes location>", () => {
```

Comment:
Removed the other test modification in favor of a dedicated `useLocation` test here